go.mod

```diff
@@ -5,25 +5,25 @@ go 1.21
 toolchain go1.21.5
 
 require (
-	github.com/ciscoecosystem/aci-go-client/v2 v2.30.0
+	github.com/ciscoecosystem/aci-go-client/v2 v2.31.0
```

Comment:
Maybe the commit message should also mention this update?

Reply:
I do agree with @samiib. We should add this change in the commit message.

Reply:
Done!